requirements.txt

```diff
@@ -39,4 +39,9 @@ svg2rlg==0.3
 typing==3.6.6             # via sphinx
 urllib3==1.24.1           # via requests
 webencodings==0.5.1       # via html5lib
+smartypants==2.0.1
```

Comment:
This needs to go in `requirements.in` I believe?

Reply:
Eek, what the heck is pip-compile? It doesn't even seem to exist anymore.

Reply:
https://github.com/jazzband/pip-tools

Reply:
Done.